index.html

```diff
@@ -1,33 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="icon" type="image/x-icon" href="/assets/meister-favicon.jpg" />
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;600&family=Titillium+Web:wght@200;400;600&display=swap"
+        />
+        <link rel="stylesheet" href="/styles/reset.css" />
+        <link rel="stylesheet" href="/styles/global.css" />
+        <title>About Meister</title>
+    </head>
 
-<head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="icon" type="image/x-icon" href="/assets/alchemy-favicon.png" />
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link rel="stylesheet"
-        href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;600&family=Titillium+Web:wght@200;400;600&display=swap">
-    <link rel="stylesheet" href="/styles/reset.css" />
-    <link rel="stylesheet" href="/styles/global.css" />
-    <title>Web App</title>
-</head>
+    <body>
+        <header>
+            <h1>Meister: The Abject Horror</h1>
+        </header>
 
-<body>
-    <header>
-        <img src="/assets/html_css_js.png">
-        <h1>Web App</h1>
-    </header>
+        <main>
+            <h3>Just how much terror can one beast sow when they weigh only six pounds?</h3>
+            <img src="./assets/meister.jpg" alt="Meister" />
+            <p>
+                Meister is a 10 year old toy pomeranian that somehow contains the wrath and ire of
+                an Olde God in a frame smaller than a football. What he lacks in teeth he makes up
+                for in peculiar aromas and pure gumption. If you fall victim to the siren call of
+                his outrageously boopable snoot you are beholden to deliver belly scritches until he
+                inevitably gets distracted.
+            </p>
+            <h4>Hobbies include:</h4>
+            <ul>
+                <li>Frequently terrorizing an incredibly docile cat twice his size</li>
+                <br />
+                <li>Barking at anyone who comes to the door so hard that he farts</li>
+                <br />
+                <li>Violently defending his food while constantly pining for the cat's</li>
+                <br />
```

Comment:
You don't need a <br> tag inside a list, the <ul> will take care of new lines for you.

Also, you probably don't need <br> tags very much at all - look up the difference between block-level elements and inline elements and use that dictate where new lines are happening (TL;DR block-level elements like <p> start on a new line and clear the space around them whereas inline elements like <span> get nestled next to other elements on the same line)